app/Models/ImportMeta.php

```diff
@@ -50,4 +50,9 @@ public function user()
     {
         return $this->belongsTo(User::class)->withDefault();
     }
+
+    public function error()
+    {
+        return $this->hasOne(ImportFailure::class, 'import_id');
```

Comment:
don't know where else to put this comment, but I noticed that the `ImportMetaSeeder` will still create `ImportMeta` entries with status `failed`, but doesn't seed any `ImportFailure`. This way, both the "Import Status" page and the API endpoint will still show certain imports as "failed", but without an error line.

Reply:
OH! Nice!!! I was actually thinking about it on friday and then completely forgot about it, well caught, thank you!